Packs/CommonScripts/Scripts/script-FailedInstances.yml

```diff
@@ -26,26 +28,49 @@ script: |
               HumanReadable: 'done testing **' + m + '**:\n' + res[0].Contents,
               ContentsFormat: formats.markdown
           }])});
+          if (res[0].Type === entryTypes.error) {
+              countFailed++;
+          }
+          else {
+              countSuccess++;
+          }
 
           if (res[0].Type === entryTypes.error) {
-              failedInstances.push({instance: m, brand: all[m].brand, category: all[m].category, information: res[0].Contents });
+              failedInstances.push({instance: m, brand: all[m].brand, category: all[m].category, information: res[0].Contents, status: 'failed' });
+          }
+          else {
+              instances.push({instance: m, brand: all[m].brand, category: all[m].category, information: 'succeed', status: 'good' });
```

Comment:
lets change the terminology from `good` to `success` and from `failed` to `failure`